### PR TITLE
[Merged by Bors] - TWIF 51

### PR DESCRIPTION
**Pull request by drc-infinyon**
@ajhunyady / @tjtelan please review and approve TWIF 51

Rendered:
image

### CODE REVIEW

File: content/news/this-week-in-fluvio-0051.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+For this edition we have more details on **Community Contribution** 
+
+## Community update
+[Carson Rajcan](https://github.com/crajcan) presented to our team a really cool contribution to the Fluvio OSS.
+
+In the project Carson developed the functionality to apply Smart Module transformation for Producer - [PR #3014](crecdfhttps://github.com/infinyon/fluvio/pull/3014)
+
+Here is Carson's experience...
+
+## The Problem
```
Comment (ajhunyady) [inline content/news/this-week-in-fluvio-0051.md:20]: We should add a bit of context on why SmartModules are useful on the producer side. 
"Shaping  the data before it enters the topic, it spares consuming services from this burden." For example, an upstream devices does not conform new data formats, or you prefer to convert Avro to JSON for downstream processing.

File: content/news/this-week-in-fluvio-0051.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+## Why This Problem?
+
+While investigating another issue, I had recently gained some insight into how the Stream Processing Unit handles Producer requests, as well as how it uses the Fluvio Smart Engine to transform records for the Consumer. Critically, I noticed that the Smart Engine was well encapsulated -- it was going to be easy to repurpose. Also that the Producer request handler was well organized, it wasn't going to be a nightmare to plug in some additional functionality.
+
+## The Workflow
+
+Where was I going to start? Well, TDD is my friend. There were plenty of test cases for Consumer Smart Module transformations that used the CLI. All I had to do was move the Smart Module options from the Consumer commands over to the Producer commands:
+
+```
+echo "foobar@test.com" | fluvio produce emails
```
Comment (ajhunyady) [inline content/news/this-week-in-fluvio-0051.md:33]: foobar@test.com -> FooBar@test.com

Then add a result to show
foobar@test.com

Thant way it's obvious what the SmartModule purpose is.

File: content/news/this-week-in-fluvio-0051.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+fluvio consume emails -B -d --smartmodule downcase
+```
+
+becomes:
+
+```
+echo "foobar@test.com" | fluvio produce emails --smartmodule downcase
```
Comment (ajhunyady) [inline content/news/this-week-in-fluvio-0051.md:40]: Same as ☝️

**Review by ajhunyady — COMMENTED** [review]
Pretty good newsletter, I enjoyed reading it.
I have a couple of comments inline.

File: content/news/this-week-in-fluvio-0051.md (hunk 1 above)
Comment (tjtelan) [reply content/news/this-week-in-fluvio-0051.md:20]: I incorporated but kept it simple to keep close to Carson's voice

**Review by tjtelan — APPROVED** [review]
This was well written! I also integrated AJ's feedback, which I think leads stronger into the rest of the story.

I kept my edits minimal in attempt to preserve Carson's perspective and voice, but also made minor formatting and branding edits (`Smart Module` -> `SmartModule`) just so it was easier (for me) to read too

bors r+

**Comment by tjtelan** [comment]
Fixed the numbered lists. Plus corrected a small technical inaccuracy, and added links to relevant docs pages

bors r+

**Pull request closed without merging**